Branch-SDK/src/main/java/io/branch/referral/PrefHelper.java

```diff
@@ -403,23 +403,23 @@ public String getBranchKey() {
     }
     
     /**
-     * <p>Sets the {@link android.os.Build#FINGERPRINT} value of the current OS build, on the current device,
+     * <p>Sets the {@link android.os.Build#RANDOMIZEDDEVICETOKEN} value of the current OS build, on the current device,
```

Comment:
I don't think this comment is valid anymore since android.os.Build.FINGERPRINT is not used and the value is from the service